src/SearchResults.js

```diff
@@ -14,14 +15,29 @@ export default function Search({
   fetchSearchResults,
   showHeader = false,
 }) {
+  // eslint-disable-next-line camelcase
+  const page_size = 10;
+  const [
+    searchOptionsForSearchResults,
+    setSearchOptionsForSearchResults,
+  ] = useState({
+    ...SOUND_LIST_QUERY_PARAMS,
+    page_size,
```

Comment:
```suggestion
    page_size: limit,
```

Reply:
f278cceecfb0f67ccda510bb4c29a27da72c8554 makes all of the necessary changes: it renames the variable denoting the number of results per page from `page_size` to `limit`, making changes in the three places it's necessary, and it removes the two occurrences of `// eslint-disable-next-line camelcase` because we no longer need them now that we are following JavaScript's camelcase convention.